vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -116,6 +118,13 @@ def fused_moe_kernel(
     off_experts = tl.load(expert_ids_ptr + pid_m)
     b_ptrs = b_ptr + off_experts * stride_be + (offs_k[:, None] * stride_bk +
                                                 offs_bn[None, :] * stride_bn)
+
+    if use_int8_w8a8:  # only support static per expert input activation whole tensor quantization now
```

Comment:
Unfortunately I don't think per-tensor scales for activations will be accurate enough for int8. Do you think dynamic per-token quantization could be implemented?

Reply:
yeah, that could be done but I'm a bit confused on the llm-compressor recipe setup, do you have an example recipe yml file to correctly set it (without using the class to config it in code)?